webpack/assets/javascripts/foreman_tools.js

```diff
@@ -0,0 +1,14 @@
+export function showSpinner(){
```

Comment:
I have tested overriding this function in plugin.
It did work, since all in all `fTools` is a simple JS object that can be overwritten as you want.
I have added a reference to my own js file served by the theme, using deface. Then I have added the following code to it:
``` javascript
fTools.showSpinner = function showSpinner2() { console.info("new spinner"); $("#turbolinks-progress").show(); }
```
Now the system started logging "new spinner" each time `showSpinner` was called.